root/resources/views/calender.blade.php

```diff
@@ -0,0 +1,46 @@
+<html>
+
+<head>
+    <meta charset='utf-8' />
+</head>
+
+<body>
+    {{-- フォーム --}}
+    <form method="POST">
+        @csrf
+        <textarea rows="10" cols="50" name="sentence">{{ isset($sentence) ? $sentence : '' }}</textarea>
+        @error('sentence')
+            <div>{{ $message }}</div>
+        @enderror
+
+        <button type="submit">ChatGPT</button>
+    </form>
+
+    {{-- 結果 --}}
+    {{ isset($chatResponse) ? $chatResponse : '' }}
```

Comment:
[NITS]
NULL合体演算子を使うとシンプルに書けます。

```suggestion
    {{ $chatResponse ?? '' }}
```

Null 合体演算子
<https://www.php.net/manual/ja/migration70.new-features.php#migration70.new-features.null-coalesce-op>